examples/ruby-single-user/app.rb

```diff
@@ -5,14 +5,14 @@
 CONSUMER_SECRET     = 'YOUR_CONSUMER_SECRET'
 
 # Get these from http://accesstoken.io
-ACCESS_TOKEN_KEY    = 'YOUR_ACCCESS_TOKEN_KEY'
-ACCESS_TOKEN_SECRET = 'YOUR_ACCESSS_TOKEN_SECRET'
+ACCESS_TOKEN_KEY    = 'YOUR_ACCESS_TOKEN_KEY'
+ACCESS_TOKEN_SECRET = 'YOUR_ACCESS_TOKEN_SECRET'
 
-consumer = OAuth::Consumer.new(CONSUMER_KEY, CONSUMER_SECRET, site: 'https://services.planningcenteronline.com')
+consumer = OAuth::Consumer.new(CONSUMER_KEY, CONSUMER_SECRET, site: 'https://planningcenteronline.com')
```

Comment:
`https://services.planningcenteronline.com` is the correct endpoint here. In the future, the domain apex will likely redirect to `services.`.